vizro-core/src/vizro/models/_action/_action.py

```diff
@@ -421,7 +421,7 @@ class Action(_BaseAction):
     # now specifies a user defined action in YAML (ok if not possible initially since it's not already) - could just
     # enable class-based one? Presumably import_path is no longer relevant though.
     function: Annotated[  # type: ignore[misc, assignment]
-        SkipJsonSchema[Union[CapturedCallable, export_data, filter_interaction]],
+        SkipJsonSchema[Union[CapturedCallable, collapse_expand_containers, export_data, filter_interaction]],
```

Comment:
No need to add it here - this is only needed for legacy actions that existed before the new `_AbstractAction` system. We never expect a user to do `actions=vm.Action(function=collapse_expand_containers)` - they would always be using the new system `actions=collapse_expand_containers()` from the start.